tests/integration/test_charm.py

```diff
@@ -4,21 +4,98 @@
 import yaml
 
 import pytest
+import requests
+from lightkube.core.client import Client
+from lightkube.resources.rbac_authorization_v1 import Role
+from lightkube.models.rbac_v1 import PolicyRule
+import time
+from pytest_operator.plugin import OpsTest
 
 log = logging.getLogger(__name__)
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+ISTIO_GATEWAY_ADDRESS = "http://10.64.140.43.nip.io"
```

Comment:
We should probably get this IP dynamically, we can't be sure that everyone running these tests is defining the same IP range for their loadbalancer